tests/ci/sentence_transformers.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+
+python -m pip install --upgrade pip
+python -m pip install /root/workspace/optimum-habana[tests]
+cd /root/workspace/sentence-transformers/tests
+python -m pip install ..
+python -m pytest test_compute_embeddings.py
+cd /root/workspace/optimum-habana/tests
+python -m pytest test_sentence_transformers.py
```

Comment:
can we add test for all py files except training related here? https://github.com/UKPLab/sentence-transformers/tree/master/tests

Reply:
also, this test is only for perf. can we details for accuracy test for slow test?

Reply:
@libinta Some of these tests fail, I'm only including the ones that pass. We'll investigate later the issues with the failing tests.

@sidgog Not sure what you mean here, accuracy is tested in the tests called at line 7 of this script.